### PR TITLE
Revert the some changes #284

### PR DESCRIPTION
**Pull request by heejin-kim**

### CODE REVIEW

File: apps/netutils/webserver/http_client.c [hunk 1]
```diff
@@ -262,19 +290,20 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url, char **bo
...
 						HTTP_LOGE("Error: Cannot find body\n");
+						process_finish = true;
+						read_finish = true;
 						break;
 					}
 
 					/* calculate chunked size */
 					for (i = 1; sentence_end - i >= len->sentence_start; ++i) {
-						cha = (int)*(buf + sentence_end - i);
+						cha = (int) * (buf + sentence_end - i);
```
Comment (sunghan-chang) [inline apps/netutils/webserver/http_client.c]: This is wrong. Dont add spaces around *.

**Comment by sunghan-chang** [comment]
@heejin-kim What is this? This has a same title with #284, but changes are reverting #284.

File: apps/netutils/webserver/http_client.c (hunk 1 above)
Comment (heejin-kim) [inline apps/netutils/webserver/http_client.c]: I will fix it

**Comment by heejin-kim** [comment]
@sunghan-chang To synchronize with Samsung github, change those code. Also it's not same commit. #284 : added code related to 'valid' in http_handle_file function #297 :  revert some code to sync
I got a bit confused, i made a mistake. Sorry.

**Comment by sunghan-chang** [comment]
@heejin-kim You had some mistakes at #284 so that you modify (revert) some changes from #284. Right?
From your description on commit message, I can not get any information. either #284. You need to fill commit messages to know what you modify and what is purpose. This is not a internal repository.

**Pull request merged**